### ISSUE TITLE
Make the User Info API available through the SDK

### ISSUE THREAD

**Issue by spiftire** [vippsas/deno-sdk#75]
**Is your feature request related to a problem? Please describe.** 
I need to collect user information after users have signed up to a subscription. I now have to create a separate http request to handle this. I would love to use the existing SDK to achieve the same goal.

**Describe the solution you'd like** 
```ts
const client = new VippsClient({...options})
const result = await client.userInfo.get(subscription.sub)
if(result.ok) {
  const {email, name, phoneNumber} = result
  // do something with the result
}
```

**Describe alternatives you've considered** 
Now I am using a custom axios client to make the request.

**Comment by tomas-zijdemans** [comment]
Hi @spiftire, thanks for reporting.

This was available in an earlier version, but I removed since it was unused. I'm happy to re-add it.

**Comment by spiftire** [comment]
Thank you! That would be much appreciated.

**Comment by danielmartinsen** [comment]
+1 on this - we used the userinfo API, but had to write a temporary fetch function after the update

**Comment by mariuswatz** [comment]
Yes, please! Will have to roll my own for a project launching in the next few days..

**Comment by spiftire** [comment]
Any progress regarding this?
